src/pages/pet/components/observations/observations.module.scss

```diff
@@ -17,5 +17,6 @@ $dark-grey: #8E8E93;
 }
 .plainText {
   font-size: 16px;
-  margin-bottom: 15px ;
+  margin-bottom: 15px;
+  margin-top: 6px;
 }
```

Comment:
nit: Don't forget to add empty line to the end of the file

Reply:
+